Doc/using/cmdline.rst

```diff
@@ -316,12 +318,28 @@ Miscellaneous options
    * ``python -c code`` and ``python`` (REPL) command lines: Don't prepend an
      empty string, which means the current working directory.
 
-   See also the :envvar:`PYTHONSAFEPATH` environment variable, and :option:`-E`
+   See also the :envvar:`PYTHONSAFEPATH` environment variable, and :option:`-p`
    and :option:`-I` (isolated) options.
 
    .. versionadded:: 3.11
 
 
+.. cmdoption:: -p
+
+   Prepend a potentially unsafe path to :data:`sys.path`: opposite of the
```

Comment:
```suggestion
   Prepend a potentially unsafe path to :data:`sys.path`; opposite of the
```

A colon doesn't really make syntactic or semantic sense here; a semicolon is the closest match to the existing text, but there are several other ways to rephrase this if desired.